pennylane_lightning_kokkos/src/tests/Test_Backend_info.cpp

```diff
@@ -0,0 +1,135 @@
+#include "Bindings.hpp"
+#include "Error.hpp"
+#include "StateVectorKokkos.hpp"
+#include "TestHelpers.hpp"
+#include <algorithm>
+#include <cassert>
+#include <catch2/catch.hpp>
+#include <cctype>
```

Comment:
Is this header needed?

Reply:
yes, we need it since we call `isalpha()` in our tests.